reference-architecture/common/README.md

```diff
@@ -0,0 +1,2 @@
+# The Reference Architecture Common Playbooks and Roles
```

Comment:
Why not move these to the root of the repo?

playbooks/<playbook>
roles/<role>